plugins/services/src/js/reducers/serviceForm/JSONReducers/Volumes.js

```diff
@@ -21,6 +22,20 @@ const mapLocalVolumes = function(volume) {
   };
 };
 
+const mapExternalVolumes = function(volume) {
+  if (
+    this.docker &&
+    this.dockerType === "DOCKER" &&
```

Comment:
Does this check still make sense since we've merged UCR and MESOS? Feels weird to double check on `dockerType` (runtime type) and on a boolean `docker`